src/blog/tanstack-table-v9-taking-form.md

```
@@ -0,0 +1,408 @@
+
+- State management that follows React's rules, plays well with the Compiler, and works well with all javascript frameworks
+- Granular re-rendering via selectors and `Subscribe`
+- Better memory and cpu usage for large tables
+- Tree-shakable features, so the bundle reflects what you actually use, giving TanStack Table room to grow
+- Custom features that can plug into the same system as built-in features
+- `createTableHook` and `tableOptions`, so reusable table code is much easier to organize
+- Devtools that exist and are actually supported
+
+We think we are mostly done with the foundational changes for V9 in terms of state management, tree-shakability, etc. Now, while we are in beta, we have some good opportunities to make some final needed breaking changes and fixes to the features themselves before we go stable. We hope that this will not be as long of a beta process as some of the other TanStack libraries, but it depends on the feedback we get from the community and how much community help we can get to test and fix the issues that we find.
+
+If you have the stomach for a beta, I would love your help kicking the tires. You can find the V9 docs [here](/table/beta).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)